### PR TITLE
Remove Docker

### PR DESCRIPTION
**Pull request by tobias-urdin**
Run testing directly on the Ubuntu node to simply
the logic and make it easier to test and see output.

### CODE REVIEW

**Comment by jd** [comment]
That's going to increase the CI runtime quite a bit though :(

**Comment by tobias-urdin** [comment]
Yeah :( just testing though for now

**Comment by tobias-urdin** [comment]
It's even worse waiting 5+ hours for Travis CI to run the tests since there is 6.5k open source jobs in queue :p

**Comment by tobias-urdin** [comment]
@mergifyio rebase

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +17,10 @@ env:
   - TARGET: py36-postgresql
 
 before_script:
+  - ulimit -a
```
Comment (mrunge) [inline .travis.yml:20]: missing a sudo?

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +17,10 @@ env:
...
+  - ls -l /etc/security/*.conf
+  - cat /etc/security/limits.conf
+  - ulimit -n 50000
```
Comment (mrunge) [inline .travis.yml:23]: then same here

**Review by mrunge — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (tobias-urdin) [reply .travis.yml:20]: nope has nothing to do with it

Comment (tobias-urdin) [reply .travis.yml:20]: the limit is enforced by travis-ci

**Comment by tobias-urdin** [comment]
will be handled another way

**Pull request closed without merging**